<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YTMusic Deleter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="removeLibraryButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Library</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="consoleTextArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>721</width>
      <height>271</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteUploadsButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Uploads</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="consoleLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Console Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="authIndicator">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unauthenticated</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
